src/VisionModel/AlexNet.jl

```diff
@@ -0,0 +1,30 @@
+struct AlexNet <: VisionModel{ImageNet.ImageNet1k}
```

Comment:
I would prefer not to use structs to wrap the models. It doesn't gain us any functionality, because there's no value added by dispatching on the type `AlexNet`. It just adds the trouble of having to do `m.layers` to access the model, making it annoying to swap this model out with an arbitrary Flux model in some code.

We also don't need to inherit from `VisionModel{ImageNet.ImageNet1k}` (not sure where this abstract type is defined). Again, there is little value in being able to dispatch on subtypes of `VisionModel{ImageNet.ImageNet1k}`.

Reply:
Alright! I wrote that to get an opinion if to include that abstract type or not.